x-pack/plugins/transform/public/app/sections/transform_management/components/action_delete/delete_action_modal.tsx

```diff
@@ -47,6 +51,19 @@ export const DeleteActionModal: FC<DeleteAction> = ({
   const bulkDeleteModalContent = (
     <>
       <EuiFlexGroup direction="column" gutterSize="none">
+        {hasManagedTransforms ? (
+          <p>
+            <>
+              <EuiText>
+                <FormattedMessage
+                  id="xpack.transform.transformList.deleteManagedBulkTransformsDescription"
+                  defaultMessage="At least one of these transforms was deployed as part of a module; deleting them might impact other parts of the product."
```

Comment:
Transforms are not part of modules, so we should definitely drop the term 'module' here, and say something like `At least one of these transforms were provided by Elastic...`

Reply:
Updated here https://github.com/elastic/kibana/pull/122305/commits/427be5b2ff13c2453e9b03611547f978baa19cbc